storage/object/quickstart.mdx

```diff
@@ -53,8 +53,6 @@ To get started with Object Storage, you must first create a bucket. Objects are
       - Region: `nl-ams`
     - Paris, France: 
       - Region: `fr-par`
-    - Warsaw, Poland: 
-      - Region: `pl-waw`
```

Comment:
Object Storage is available in pl-waw. 😄

Reply:
It has to be specified here: https://github.com/scaleway/docs-content/pull/890/files#diff-fbcd41286715e50712518a5263d0e38c47971a609352df184fcf439126a903a4R92 (Glacier only available in `fr-par` and `nl-ams`)

Reply:
ah yeah, sorry. my bad! it's fixed